packages/next/src/server/dev/turbopack-utils.ts

```diff
@@ -563,7 +579,7 @@ async function writeLoadableManifest(
 }
 
 export async function writeManifests(
-  opts: SetupOpts,
+  rewrites: SetupOpts['fsChecker']['rewrites'],
   distDir: string,
   buildManifests: BuildManifests,
   appBuildManifests: AppBuildManifests,
```

Comment:
Could also be an options object

Reply:
Opened https://github.com/vercel/next.js/pull/61996 to handle it 👍